Tests/Application/Kernel.php

```diff
@@ -1,52 +1,38 @@
 <?php
 
+declare(strict_types=1);
+
 /*
  * This file is part of Sulu.
  *
- * (c) MASSIVE ART WebServices GmbH
+ * (c) Sulu GmbH
  *
  * This source file is subject to the MIT license that is bundled
  * with this source code in the file LICENSE.
  */
 
+namespace Sulu\Bundle\CommentBundle\Tests\Application;
+
 use Sulu\Bundle\CommentBundle\SuluCommentBundle;
 use Sulu\Bundle\TestBundle\Kernel\SuluTestKernel;
+use Symfony\Bundle\SecurityBundle\SecurityBundle;
 use Symfony\Component\Config\Loader\LoaderInterface;
 
-class AppKernel extends SuluTestKernel
+class Kernel extends SuluTestKernel
 {
-    /**
-     * {@inheritdoc}
-     */
-    public function __construct($environment, $debug, $suluContext = self::CONTEXT_ADMIN)
-    {
-        $this->name = $suluContext;
-
-        parent::__construct($environment, $debug, $suluContext);
-    }
-
-    /**
-     * {@inheritdoc}
-     */
     public function registerBundles()
     {
-        $bundles = array_merge(
-            [
-                new SuluCommentBundle(),
-            ],
-            parent::registerBundles()
-        );
-
-        if (self::CONTEXT_ADMIN !== $this->getContext()) {
+        $bundles = parent::registerBundles();
+        $bundles[] = new SuluCommentBundle();
+
+        if (SuluTestKernel::CONTEXT_WEBSITE === $this->getContext()) {
             $bundles[] = new \FOS\RestBundle\FOSRestBundle();
+            $bundles[] = new SecurityBundle();
```

Comment:
Do we really need this on the website context?
Is this documented or is it just needed for tests?

Reply:
it is only neccesary to check if the user isset for a newly created comment